### PR TITLE
css of summary screen

### PR DESCRIPTION
**Pull request by abishekTa-egov**
Changes is css of summary screen

## Release Notes

- **New Features**
	- Updated stylesheet for the health UI component to version 0.1.18, potentially introducing new styles.
	- Added new CSS classes for improved layout and presentation in various components.

- **Improvements**
	- Enhanced rendering logic in several components for better visual separation and user interaction.
	- Simplified button styling across multiple components while maintaining functionality.

- **Bug Fixes**
	- Streamlined state management and rendering logic in components to improve user experience.

- **Documentation**
	- Updated imports and component structures for better clarity and organization.

### CODE REVIEW

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SummaryScreen.js [hunk 1]
```diff
@@ -234,7 +234,14 @@ const SummaryScreen = ({ props: customProps }) => {
     },
   };
 
-  return <ViewComposer data={data} />;
+  return (
+  <>
+  <Header>
+    {t("SUMMARY_SCREEN")}
+  </Header>
+  <ViewComposer data={data} />;
```
[the remark this answers is not in the document]
Comment (ashish-egov) [reply health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SummaryScreen.js:242]: @abishekTa-egov Check this

File: health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss [hunk 2]
```diff
@@ -248,3 +248,17 @@ a{
...
+  display: -ms-flexbox;
+  display: flex;
+  margin-top: 1.5rem;
+}
+
+.header-style{
+  margin-top: 0;
+  font-size: 1.8rem;
+  color: #0B4B66 !important;
+}
```
[the remark this answers is not in the document]
Comment (ashish-egov) [reply health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss:264]: @abishekTa-egov  check this

File: health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SummaryScreen.js [hunk 3]
```diff
@@ -3,7 +3,7 @@ import { useTranslation } from "react-i18next";
 import { useHistory } from "react-router-dom";
 import { EditIcon, ViewComposer } from "@egovernments/digit-ui-react-components";
 import { Loader } from "@egovernments/digit-ui-components";
-
+import { Header } from "@egovernments/digit-ui-components";
```
Comment (ashish-egov) [inline health/micro-ui/web/micro-ui-internals/packages/modules/microplan/src/pages/employee/SummaryScreen.js:6]: @abishekTa-egov Consolidated imports from "@egovernments/digit-ui-components" into a single line for cleaner code.

File: health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss (hunk 2 above)
[the remark this answers is not in the document]
Comment (abishekTa-egov) [reply health/micro-ui/web/micro-ui-internals/packages/css/src/components/microplan.scss:264]: It has to be important else its overwritten

**Review by ashish-egov — APPROVED** [review]

**Review by nipunarora-eGov — APPROVED** [review]

**Pull request merged**